test/e2e-go/features/transactions/group_test.go

```diff
@@ -202,3 +203,80 @@ func TestGroupTransactionsDifferentSizes(t *testing.T) {
 		a.Error(err)
 	}
 }
+
+func TestGroupTransactionsSubmission(t *testing.T) {
+	t.Parallel()
+	a := require.New(t)
+
+	var fixture fixtures.RestClientFixture
+	fixture.Setup(t, filepath.Join("nettemplates", "TwoNodes50EachFuture.json"))
+	defer fixture.Shutdown()
+
+	client := fixture.LibGoalClient
+	accountList, err := fixture.GetWalletsSortedByBalance()
+	a.NoError(err)
+	account0 := accountList[0].Address
+	goodGroupSizes := []int{1, 2, 16}
+	exceedGroupSize := 17
```

Comment:
avoid using the number 16 and 17 directly. instead, query the consensus params and use the magic numbers from there.

Reply:
done